CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
+- Added `LumpedResistor` and`LumpedPort`, which make up the new `TerminalComponentModeler` in the `smatrix` plugin
```

Comment:
space after `and`. Also, I think some mention of the fact that `Simulation` accepts `lumped_elements` could be useful.